index.html

```diff
@@ -1,30 +1,33 @@
 <!DOCTYPE HTML>
 <html>
-	<head>
-		<title> Github - Testes </title>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-		<link rel="stylesheet" href="assets/css/main.css" />
-		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
-	</head>
-	<body class="is-preload">
-
-		<!-- Header -->
-			<header id="header">
-				<h1>Big Picture</h1>
-				<nav>
-					<ul>
-						<li><a href="#intro">Home</a></li>
-						<li><a href="#one">Juliana</a></li>
-            			<li><a href="#two">Thiago</a></li>
-						<li><a href="#three">Renata</a></li>
-					</ul>
-				</nav>
-			</header>
-			<footer>
-				<a href="#one" class="button style2 down">More</a>
-			</footer>
-		</div>
+
+<head>
+	<title> Github - Testes </title>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+	<link rel="stylesheet" href="assets/css/main.css" />
+	<noscript>
+		<link rel="stylesheet" href="assets/css/noscript.css" /></noscript>

```

Comment:
O que acha de mantermos a indentação desta forma?
```suggestion
	<noscript>
		<link rel="stylesheet" href="assets/css/noscript.css" />
	</noscript>

```

Reply:
Faz sentido, obrigada pela sugestão Bruno.

Identação será ajustada no commit seguinte.